### PR TITLE
add stepper component

### PR DESCRIPTION
**Pull request by mazmassa**
We are adding the `<Stepper>` component.

Screenshot 2023-05-11 at 15 56 38

⚠️ We are not following the exact design for that, for now.

### CODE REVIEW

**Review by pivilartisant — COMMENTED** [review]
Hey mario I have two questions : 
- Is it possible to set the steps text under the icon like in design ? 
- Right now on storybook it's static but I see that you have written functions, will the step progression be implemented next ?

**Comment by mazmassa** [comment]
> Hey mario I have two questions :
> 
> * Is it possible to set the steps text under the icon like in design ?
> * Right now on storybook it's static but I see that you have written functions, will the step progression be implemented next ?


My bad to no mention in the description: 

> * Is it possible to set the steps text under the icon like in design ?
No, this will be a second version because it demands more time to finish. I have a previous version already done. But in future we update that.

>Right now on storybook it's static but I see that you have written functions, will the step progression be implemented next 
Sure!

File: src/components/Stepper/Stepper.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      <FiCircle className="w-1 h-1 text-f-primary fill-i-primary" />
+    </span>
+  );
+}
+
+function StepNext() {
+  return (
+    <span
+      data-testid="stepper-step-next"
+      className={`${baseStepClass} border border-1 border-c-default bg-transparent`}
```
[the remark this answers is not in the document]
Comment (mazmassa) [reply src/components/Stepper/Stepper.tsx]: Definitely!

Removed!

**Review by maxime-carabina — APPROVED** [review]
LGTM

**Review by pivilartisant — APPROVED** [review]

**Pull request merged**